Makefile

```
@@ -16,3 +16,6 @@ deploy_contract_anvil:
 	--private-key 0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6 \
 	contract/script/MinaBridge.s.sol:MinaBridgeDeployer
 # deploy_contract_anvil uses Anvil wallet 9, same as Aligned for deploying its contracts.
+
+verify_account_inclusion:
+	@cargo run --manifest-path account_inclusion/Cargo.toml --release -- $(public_key)
```

To keep consistency:

```suggestion
	@cargo run --manifest-path account_inclusion/Cargo.toml --release -- ${PUBLIC_KEY}
```